static/app/utils/metrics/metricsRequest.tsx

```diff
@@ -47,11 +53,16 @@ type Props = DefaultProps & {
   limit?: number;
   interval?: string;
   isDisabled?: boolean;
+  /**
+   * Transform the response data to be something ingestible by GridEditable tables
+   */
+  includeTabularData?: boolean;
```

Comment:
MetricsRequest will now be able to return not only raw responses but also data pre-massaged for our tables.